src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -131,6 +131,16 @@ public static String sanitizeForHtml(String str) {
     }
 
     /**
+     * Escapes HTML tag safely. This function can be applied multiple times.
+     */
+    public static String sanitizeForHtmlTag(String str) {
+        if (str == null) {
+            return null;
+        }
+        return str.replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;");
```

Comment:
That's because DOM Document will unsanitize the text content when converting HTML output and page source into standard HTML. 
For example: we have a page source like this:
`<p>This &amp;lt;/p&amp;gt; is sanitized version of &lt;/p&gt;</p>`
will be displayed as <p>This &amp;lt;/p&amp;gt; is sanitized version of &lt;/p&gt;</p>
So the text content here is `This &amp;lt;/p&amp;gt; is sanitized version of &lt;/p&gt;`. DOM Document will unsanitize it to `This &lt;/p&gt; is sanitized version of </p>`.
If we just sanitize '<' and '>' only, it will become `This &lt;/p&gt; is sanitized version of &lt;/p&gt;` and will be displayed as 
This &lt;/p&gt; is sanitized version of &lt;/p&gt;
That's not the original content.